catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -228,6 +235,8 @@
 
     private static final XMLUtils XML_UTILS = XMLUtils.getInstance();
 
+    private static final int DEFAULT_PAGE_SIZE = 20;
```

Comment:
Why 20? Do we have any metrics to attempt to optimize this to some degree? 20 seems small to me but I have no physical evidence to back that up.